dydx/dydxPresenters/dydxPresenters/_v4/MarketInfo/Components/dydxMarketPositionViewPresenter.swift

```diff
@@ -78,69 +78,78 @@ class dydxMarketPositionViewPresenter: HostedViewPresenter<dydxMarketPositionVie
+                let stepSizeDecimals = market.configs?.stepSizeDecimals?.intValue ?? 0
+                if takeProfitOrders.count > 1 {
+                    viewModel?.takeProfitStatusViewModel = .init(
+                        triggerSide: .takeProfit,
+                        triggerPriceText: DataLocalizer.shared?.localize(path: "APP.TRADE.MULTIPLE_ARROW", params: nil),
+                        action: routeToOrdersAction)
+                } else if let takeProfitOrder = takeProfitOrders.first, let positionSize = position.size?.current?.doubleValue.magnitude {
+                    let orderSize = takeProfitOrder.size.magnitude
+                    viewModel?.takeProfitStatusViewModel = .init(
+                        triggerSide: .takeProfit,
+                        triggerPriceText: dydxFormatter.shared.dollar(number: takeProfitOrder.triggerPrice?.doubleValue, digits: decimalDigits),
+                        limitPrice: takeProfitOrder.type == .takeprofitlimit ? dydxFormatter.shared.dollar(number: takeProfitOrder.price, digits: decimalDigits) : nil,
+                        amount: positionSize == orderSize ? nil : dydxFormatter.shared.percent(number: orderSize/positionSize, digits: 2),
+                        action: routeToTakeProfitStopLossAction)
+                } else {
+                    viewModel?.takeProfitStatusViewModel = .init(
+                        triggerSide: .takeProfit,
+                        action: routeToTakeProfitStopLossAction)
+                }
+
+                if stopLossOrders.count > 1 {
+                    viewModel?.stopLossStatusViewModel = .init(
+                        triggerSide: .stopLoss,
+                        triggerPriceText: DataLocalizer.shared?.localize(path: "APP.TRADE.MULTIPLE_ARROW", params: nil),
+                        action: routeToOrdersAction)
+                } else if let stopLossOrder = stopLossOrders.first, let positionSize = position.size?.current?.doubleValue.magnitude {
+                    let orderSize = stopLossOrder.size.magnitude
+                    viewModel?.stopLossStatusViewModel = .init(
+                        triggerSide: .stopLoss,
+                        triggerPriceText: dydxFormatter.shared.dollar(number: stopLossOrder.triggerPrice?.doubleValue, digits: decimalDigits),
+                        limitPrice: stopLossOrder.type == .stoplimit ? dydxFormatter.shared.dollar(number: stopLossOrder.price, digits: decimalDigits) : nil,
+                        // don't show amount unless order size is custom
+                        amount: positionSize == orderSize ? nil : dydxFormatter.shared.percent(number: orderSize/positionSize, digits: 2),
+                        action: routeToTakeProfitStopLossAction)
+                } else {
+                    viewModel?.stopLossStatusViewModel = .init(
+                        triggerSide: .stopLoss,
+                        action: routeToTakeProfitStopLossAction)
+                }
```

Comment:
new logic for display different states for this area
![Screenshot 2024-04-29 at 4 48 06 PM](https://github.com/dydxprotocol/v4-native-ios/assets/149746839/9e109870-19f6-4ae4-8d35-d167d65cb163)